home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java

```diff
@@ -105,7 +105,7 @@ private ReadCooperation createTestCooperationWithUserEmail(String userEmail) {
     @SneakyThrows
     private ReadCooperation createTestCooperationAndUserViaInvitationWithUserEmail(String userEmail) {
         ReadCooperation readCooperation = createTestCooperationWithUserEmail(userEmail);
-        TimeUnit.MILLISECONDS.sleep(5000);
+        TimeUnit.MILLISECONDS.sleep(10000);
```

Comment:
Is it something that isn't working with the previous delay just on your local box or has everybody reported that we need to increase it?

Reply:
I just forgot to change delay:)